src/twisted/conch/test/test_ckeygen.py

```diff
@@ -172,6 +174,19 @@ def test_printFingerprintBadFingerPrintFormat(self):
             "Unsupported fingerprint format: sha-base64", em.exception.args[0]
         )
 
+    def test_printFingerprintSuffixAppended(self):
```

Comment:
Let's not accidentally turn off type-checking in the tests

```suggestion
    def test_printFingerprintSuffixAppended(self) -> None:
```

Reply:
Added return types for tests.